tests/test_filter_count.py

```diff
@@ -180,39 +278,74 @@ def test_prune_on_small_data(filter_by):
     assert not any(noise_idx)
 
 
-def test_calibrate_joint():
+@pytest.mark.parametrize(
+    "multi_label",
+    [True, False],
+)
+def test_calibrate_joint(multi_label):
```

Comment:
can we just split this into two separate tests one for multi_label, one for regular classification. The two tests share hardly anything in common, so should not be one function IMO.